apm-lambda-extension/apm/apmserver.go

```diff
@@ -0,0 +1,214 @@
+// Licensed to Elasticsearch B.V. under one or more contributor
+// license agreements. See the NOTICE file distributed with
+// this work for additional information regarding copyright
+// ownership. Elasticsearch B.V. licenses this file to you under
+// the Apache License, Version 2.0 (the "License"); you may
+// not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+package apm
+
+import (
+	"bytes"
+	"compress/gzip"
+	"context"
+	"elastic/apm-lambda-extension/extension"
+	"errors"
+	"fmt"
+	"io"
+	"math"
+	"math/rand"
+	"net/http"
+	"time"
+)
+
+// StartBackgroundApmDataForwarding Receive agent data as it comes in and post it to the APM server.
```

Comment:
```suggestion
// ForwardApmData receives agent data as it comes in and posts it to the APM server.
```

Reply:
Good catch! 

I also renamed the package :+1: